src/confirmationModal/ConfirmationModal.tsx

```diff
@@ -26,6 +27,7 @@ export interface ConfirmationModalProps extends ModalProps {
   readonly confirmLabel?: string
   readonly confirmIsLoading?: boolean
   readonly size?: ModalSize
+  readonly children?: string
```

Comment:
I agree with you, it feels a bit weird to have children as string. You may use another prop instead, i.e. `label`

Reply:
What about just 'title' ?

Reply:
@mcellier @grosbouddha To avoid break changes on a "good first issue", I would rather keep the `TheVoice` API and only change `children` type to `ReactNode`

Check it out:
https://medium.com/@martin_hotell/react-children-composition-patterns-with-typescript-56dfc8923c64

It sounds a good approach since `TheVoice` could also receive `<br />`  for exemple.

I've also updated `Title` component `children` type, since it has been declared as `string` before.